velox/row/tests/UnsafeRowFuzzTest.cpp

```diff
@@ -235,5 +293,23 @@ TEST_F(UnsafeRowFuzzTests, fast) {
       });
 }
 
+TEST_F(UnsafeRowFuzzTests, nestedMaps) {
```

Comment:
I assume this is not a fuzz test, is it? Why is it part of UnsafeRowFuzzTests?

Reply:
We don't have UnsafeRowTest, do I need to add a new File UnsafeRowTest?

Reply:
I would suggest to rename the `UnsafeRowFuzzTest` to `UnsafeRowTest`.